corpora-api.yml

```diff
@@ -493,6 +497,131 @@ components:
     visibility:
       type: string
       enum: [public,private]
+    project:
+      type: object
+        properties:
+          description:
+            type: string
+          name:
+            type: string
+          id:
+            $ref: '#/components/schemas/project_uuid'
+          s3_bucket_key:
+            type: string
+          submitter:
+            type: object
+            properties:
+              id:
+                $ref: '#/components/schemas/user_uuid'
+              email:
+                type: string
+          links:
+            type: object
+            properties:
+              protocol:
+                type: array
+                items:
+                  type: string
+              other:
+                type: array
+                items:
+                  type: string
+              raw_data:
+                type: array
+                items:
+                  type: string
+              attestation:
+                type: array
+                items:
+                  type: string
+          datasets:
```

Comment:
This is fine to start, though I expect we'll have more nuanced access patterns separating project and dataset details. For example, potentially only returning a list of `dataset_id`s here that can be further introspected on via a GET `/dataset`.

The advantage of this is reducing payload size and organizing the API, though no action required now, we can shrink the response later if it makes sense.

Reply:
I agree fully, future optimization will likely be needed. For expendiance, I focused on building something simple and expandible.